doc/chapters/integration-guidelines/10-traffic-shaping.rst

```diff
@@ -0,0 +1,72 @@
+Traffic Shaping
+===============
+
+Linux offers a very rich set of tools for managing and manipulating the transmission of packets. And
+since the host system is Linux based, those tools can be used to manage and shape traffics on Software
+Containers. Linux community is very familiar with the tools available under Linux for packet mangling and
```

Comment:
Either write "SoftwareContainer" or "containers", or "SoftwareContainer containers", but not "Software Containers".

Also: **The** Linux community